scripts/utils/check_replied.py

```
@@ -0,0 +1,310 @@
+BOT_SIGNATURES = [
+    "hypershift-jira-solve-ci[bot]",
+    "hypershift-jira-solve-ci",
+]
+
+# Text signature that appears in bot replies
+REPLY_SIGNATURE = "*AI-assisted response via Claude Code*"
+
+
+def run_gh(args: list[str]) -> Any:
+    """Run gh CLI command and return JSON output."""
+    result = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)